scrapers/infernalrestraints.yml

```diff
@@ -0,0 +1,40 @@
+name: infernalrestraints
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - infernalrestraints.com/device
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //a[@class="articleTitleText"][1]/text()
+      Date:
+        selector: //span[@class="articlePostDateText"][1]/text()
+        postProcess:
+          - parseDate: "January 02, 2006"
```

Comment:
this `          - parseDate: "January 02, 2006"` isnt needed
`January 02, 2006` is for sites that use 0 padding for single digit days eg April 01, 2020
`January 2, 2006` parses sites with April 1, 2020 instead
Both parse April 11, 2020

Reply:
fixed